cluster-scope/overlays/prod/moc/smaug/imageregistry.operator.openshift.io/configs/cluster/config_patch.yaml

```diff
@@ -1,13 +0,0 @@
----
-apiVersion: imageregistry.operator.openshift.io/v1
-kind: Config
-metadata:
-  name: cluster
-spec:
-  storage:
-    $patch: replace
-    managementState: Unmanaged
-    s3:
-      bucket: openshift-image-registry-bucket
-      regionEndpoint: "https://s3-rook-openshift-storage.apps.morty.emea.operate-first.cloud/"
```

Comment:
smaug cluster needs an alternative bucket for image registry

Reply:
bucket already exists here: https://github.com/operate-first/apps/tree/mastercluster-scope/overlays/prod/moc/smaug/objectbucketclaims/openshift-image-registry-bucket.yaml. Adding the config-patch back but instead pointing it to the existing smaug bucket.